Dockerfile.testing

```diff
@@ -7,12 +7,18 @@ LABEL maintainer OSG Software <support@opensciencegrid.org>
 
 RUN \
     yum update -y && \
-    yum install -y --enablerepo=devops-itb osg-ca-generator && \
+    yum install -y git && \
     yum install -y --enablerepo=osg-upcoming condor && \
     yum install -y python3-pip httpd mod_auth_openidc mod_ssl python3-mod_wsgi && \
-    osg-ca-generator --host && \
     yum clean all && rm -rf /var/cache/yum/*
 
+RUN \
+    git clone https://github.com/opensciencegrid/osg-ca-generator.git && \
```

Comment:
Might want to do a `cd /tmp` or something first

Reply:
Done